src/main/java/seedu/address/logic/parser/QueryPatientCommandParser.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.util.Arrays;
+
+import seedu.address.logic.commands.QueryPatientCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.PatientNameContainsKeywordsPredicate;
+
+/**
+ * Parses input arguments and creates a new FindCommand object
+ */
+public class QueryPatientCommandParser implements Parser<QueryPatientCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the FindCommand
```

Comment:
Perhaps need to update the javadocs here to match actual method.

Reply:
updated! thanks for the catch